index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/css/stylesheet.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cutive+Mono&family=Montserrat:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>My library</title>
+  </head>
+  <body class="body output">
+    <div class="main">
+      <h1 class="main-title">My Library.</h1>
+      <button  class="form-btn" id ="new-book-btn" onclick="showForm()">Add a new BOOK!</button>
+      <form  class="new-book-form d-flex" id="new-book-form">
+        <div class="m-15">
+          <label for="book-title">Book title</label><br />
+          <input type="text" class="input" id="book-title" placeholder="Title" />
+        </div>
+        <div class="d-flex form-group">
+          <div class="m-15">
+            <label for="book-pages">Number of pages</label><br />
+            <input
+              type="text"
+              class="input"
+              id="book-pages"
```

Comment:
- [ ] **OPTIONAL** Did you know that the [id](https://www.w3schools.com/html/html_id.asp) attribute is very unique! That's why it's only used on unique elements that won't be repeated again. But, since you're using classes on all the elements, you could easily target those elements in CSS and JS with only their class names using the [`querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector).